src/applications/mhv/secure-messaging/tests/e2e/fixtures/userResponse/user-cerner-all.json

```diff
@@ -612,7 +612,7 @@
   "meta": {
       "errors": [
           {
-              "externalService": "EMIS",
+              "externalService": "Vet360",
```

Comment:
What is your reasoning behind this being Vet360 and not VAProfile?

Reply:
oh, yeah, good question. It seems to be expecting `Vet360` in `src/platform/user/profile/utilities/index.js`
![Screenshot 2024-02-12 at 10 25 37 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/10993987/f8b94141-8da4-4e37-9016-73a90e460388)

And also in vets-api [here](https://github.com/department-of-veterans-affairs/vets-api/pull/15393/files/c4ee4257aa2c16329798820298c49c1f58e605f3#diff-64abab86a25a2db0f3b93876d46a4098b13d0a5b689da27c8171c2edb9cd88e5). 

🤔 I wonder if I changed _all_ of these to VAProfile plus changed that line in vets-api ^^ to VAProfile, if that would work. It makes me a little worried... but if it works, that would cut down on work for the Vet360 -> VA Profile name change work.

Reply:
Looks like you've made clarifying changes to this already!